packages/react-components/src/theming/icons.tsx

```diff
@@ -73,6 +74,7 @@ export const DEFAULT_COMPONENT_ICONS = {
   ControlButtonParticipants: <People20Filled />,
   ControlButtonScreenShareStart: <ShareScreenStart20Filled />,
   ControlButtonScreenShareStop: <ShareScreenStop20Filled />,
+  CameraSwitch: <CameraSwitch24Filled />,
```

Comment:
@prprabhu-ms @PorterNan should this be beta-only compiled?

Reply:
Makes sense. Yes, we should restrict this to beta flavor builds.